### ISSUE TITLE
Wrong TOTP is generated

### ISSUE THREAD

**Issue by colemickens** [geluk/pass-winmenu#102]
I'm not sure if this ever worked, but I'm getting the wrong TOTP generated for certain entries.

For example, `gopass totp X` gives the same (correct) TOTP consistently, while this does not. (The time/date on this machine are right too, and I'm running the gopass totp here too).

FWIW, this is the entry:

```
otpauth: //totp/AWS?secret=ZZZN3LXXXCD7V6969CJ5ZXFLLB64KXNGI25AYYYDLMC2MPABCDEF&issuer=aws&digits=6 
```

with some digits changed. This was for Amazon, also, if it helps testing, etc.

**Comment by geluk** [comment]
Can you provide an example of a valid TOTP URI for which pass-winmenu doesn't generate the right TOTP codes? The one you showed here contains invalid base32 characters, so I can't use it to test the implementation.

**Comment by colemickens** [comment]
```
Secret: colemickens/amazon2.com

login: cole.mickens@gmail.com
otpauth: //totp/AWS?secret=LJUN3L54ZCD7VLULQCJ5ZXFLLB64KXNGI25APKNDLMC2MPCGDKYA&issuer=aws&digits=6
url: https://www.amazon.com/
```

Sure, that's fine, here's the full totp uri.

**Comment by geluk** [comment]
Thanks! I think I found the cause. I've created a new test build which you can download here: https://ci.appveyor.com/api/buildjobs/nqaf3na89x28oem5/artifacts/pass-winmenu%2Fbin%2FRelease%2Fpass-winmenu.exe

Could you check if this resolves your issue?

**Comment by colemickens** [comment]
That build works with my Facebook and GitHub logins, whereas the current release does not. Whatever the fix was, I'd say shipit!

(And thank you, of course!)

**Comment by chickentest** [comment]
Hello, I'm using this program and like it very much. However where I can find the release with OTP function?
this link https://ci.appveyor.com/api/buildjobs/nqaf3na89x28oem5/artifacts/pass-winmenu%2Fbin%2FRelease%2Fpass-winmenu.exe
show me: {"message":"Artifact not found or access denied."}
Please help. I have a lot of accounts with OTP enabled...

**Comment by geluk** [comment]
Hi @chickentest, I'll publish a full release with this feature shortly. The build links I share expire after a month, I've generated a new one that you can use in the meantime: https://ci.appveyor.com/api/buildjobs/8jpr647aoeer05h6/artifacts/pass-winmenu%2Fbin%2FRelease%2Fpass-winmenu.exe

**Comment by chickentest** [comment]
Thank You! And Happy New Year!30.12.2021, 20:38, "Johan Geluk" ***@***.***>:
Hi @chickentest, I'll publish a full release with this feature shortly. The build links I share expire after a month, I've generated a new one that you can use in the meantime: https://ci.appveyor.com/api/buildjobs/8jpr647aoeer05h6/artifacts/pass-winmenu%2Fbin%2FRelease%2Fpass-winmenu.exe

**Comment by geluk** [comment]
TOTP is now available in release [v1.13](https://github.com/geluk/pass-winmenu/releases/tag/v1.13).
